pytorch_to_returnn/torch/nn/modules/module.py

```diff
@@ -757,6 +757,11 @@ def _base_get_output_shape_from_returnn(inputs_flat: List[Tensor], layer: LayerB
     # DimensionTag.get_all_dimension_tags, etc.
     # However, this is simpler, and also more consistent with get_returnn_axis_description.
 
+    def _get_unique_spatial_dim_tag_name(data, axis):
+      dim_tag = data.get_dim_tag(axis)
+      idx = data.get_axes_by_tag_name(dim_tag.description).index(axis)
+      return f"stag-single:{idx}:{dim_tag.description}"
```

Comment:
Why do you need this as a string here?
I think better would be:
```suggestion
      return dim_tag, idx
```

Reply:
And then change types, function name, and other documentation accordingly.

Reply:
E.g. the function could be named `_get_spatial_dim_tag_single` (to be consistent to "stag-single").

Reply:
It does not need to be a string, but I somehow need the index to keep the `dim_tag`s apart. I could also use a tuple `(dim_tag, idx)` as key to `dyn_size_dim_tag_to_spatial_idx_and_torch_dim` (which could possibly renamed). Do you have preferances here?

Reply:
But this is exactly what I suggested? See my suggested change above.

Reply:
Oh sorry, somehow the two additional comments did not show up before.